src/autoCollection/metrics/statsbeat/statsbeat.ts

```diff
@@ -362,7 +362,7 @@ export class Statsbeat {
                 metrics: [statsbeat],
                 properties: this._statbeatMetrics[i].properties,
             };
-            this._metricHandler.trackStatsbeatMetric(metricTelemetry);
+            // this._metricHandler.trackStatsbeatMetric(metricTelemetry);
```

Comment:
Is this just being left in as a placeholder for exporting statsbeat as a custom metric?

Reply:
That is correct, removing old code so AzureMonitorExporter is used instead